### PR TITLE
Fix bug due to poor naming of QResources attribute

### PR DESCRIPTION
**Pull request by gpetretto**
The `kwargs` attribute of `QResources` is not handled correctly by monty during serialization. 
Renaming it to `qkwargs`

### CODE REVIEW

File: src/qtoolkit/core/data_objects.py [hunk 1]
```diff
@@ -171,7 +171,7 @@ class QResources(QTKObject):
     njobs: int | None = None
     """Number of jobs in a job array."""
 
-    kwargs: dict | None = None
+    qkwargs: dict | None = None
```
Comment (davidwaroquiers) [inline src/qtoolkit/core/data_objects.py]: Thinking about the name, maybe scheduler_kwargs ? more verbose but basically it is keyword arguments to be passed to a baseschedulerIO instance right ?

File: src/qtoolkit/core/data_objects.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ class QResources(QTKObject):
...
     """Additional keyword arguments to be passed to the queue manager."""
```
Comment (davidwaroquiers) [inline src/qtoolkit/core/data_objects.py]: "to be passed to the scheduler IO object." ?

**Review by davidwaroquiers — APPROVED** [review]
Good to me except (maybe) for naming ? (see line comments) I let you decide if you think qkwargs is fine or else, it is not a super crucial point anyway.

**Comment by davidwaroquiers** [comment]
All good to me!
About the timedelta, in the end I guess you went with removing it then? (if we see a need in future, we can also add it again)
Can be merged!

**Comment by gpetretto** [comment]
Yes, I also removed the timedelta type for `time_limit`. Adding it back with a better support will be possible at a later time.

**Pull request merged**